Week1/homework/meetup/meetup.js

```diff
@@ -0,0 +1,63 @@
+
+const mysql = require('mysql');
+
+// Require the data module
+const data = require('./data.js');
+const inviteesInfo = data.inviteesInfo;
+const roomsInfo = data.roomsInfo;
+const meetingsInfo = data.meetingsInfo;
+// console.log(inviteesInfo)
+
+// Create the connection
+const connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  database : 'meetup',
+  // port : 3307
+});
+
+connection.connect();
+// Create tables
+const invitees = "CREATE TABLE invitees (invitee_no INT AUTO_INCREMENT, invitee_name VARCHAR(50), invited_by VARCHAR(50), PRIMARY KEY(invitee_no))";
+
```

Comment:
why extra newline ?
Again, try being consistent with extra spaces and indentation etc.

Reply:
OK. Done! 
I have already ensured that the file is consistent and deleted extra spaces. I will make sure to consider this feedback in the future. 
Thanks for the feedback.